tsrc/executor.py

```diff
@@ -0,0 +1,71 @@
+""" Helpers to run thing on multiple repos and collect errors """
+
+import abc
+
+from tsrc import ui
+import tsrc
+
+
+class ExecutorFailed(tsrc.Error):
+    pass
+
+
+class Actor(metaclass=abc.ABCMeta):
```

Comment:
Yeah, naming is hard 😛 

I'll try to see if there's a design pattern that matches what we do here